packages/@amazeelabs/bridge-react-router/src/index.tsx

```diff
@@ -0,0 +1,38 @@
+import type {
+  LinkType,
+  LocationProviderType,
+  useLocationType,
+} from '@amazeelabs/bridge';
+import React from 'react';
+import {
+  Link as ReactRouterLink,
+  useLocation as useReactRouterLocation,
+  useNavigate,
+} from 'react-router';
+
+export const useLocation: useLocationType = () => {
+  const location = useReactRouterLocation();
+  const navigate = useNavigate();
+
+  return [
+    {
+      pathname: location.pathname,
+      search: location.search,
+      searchParams: new URLSearchParams(location.search),
+      hash: location.hash,
+    },
+    navigate,
+  ];
+};
+
+export const Link: LinkType = ({ href, children, ...props }) => {
+  return (
+    <ReactRouterLink {...props} to={href || '/'}>
+      {children}
+    </ReactRouterLink>
+  );
+};
+
+export const LocationProvider: LocationProviderType = ({ children }) => {
+  return <>{children}</>;
+};
```

Comment:
@Leksat Some other APIs can be nice to expose. For example:
- https://reactrouter.com/api/hooks/useSearchParams (can replace some current use cases of `navigate`)
- https://reactrouter.com/api/components/PrefetchPageLinks
- https://reactrouter.com/api/components/NavLink

But that would require changes in `@amazeelabs/scalars` and `@amazeelabs/bridge-storybook`. Not sure if it's simpler than that? 🤔

Reply:
I'd say let's go with minimum for now. And once Philipp is back, you could check with him.